process/peer/process.go

```diff
@@ -0,0 +1,288 @@
+package peer
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/marshal"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+	"github.com/ElrondNetwork/elrond-go/storage"
+)
+
+// ShardedPeerAdapters represents a list of peer account adapters organized by the shard id
+//  each validator account is alocated to
+type ShardedPeerAdapters map[uint32]state.AccountsAdapter
```

Comment:
I think this shoud be private

Reply:
It is actually passed as a parameter from struct.go so wee need it exported